include/swift/SILOptimizer/Analysis/DifferentiableActivityAnalysis.h

```diff
@@ -206,6 +206,16 @@ class DifferentiableActivityInfo {
   /// Returns the activity of the given value for the given `SILAutoDiffIndices`
   /// (parameter indices and result index).
   Activity getActivity(SILValue value, const SILAutoDiffIndices &indices) const;
+
+  /// Prints activity information for the `indices` of the given `value`.
+  void dumpActivityInfo(SILValue value,
```

Comment:
How about renaming these to just `dump`?

Reply:
Ah that is better. Done.